apollo-router-scaffold/templates/base/xtask/Cargo.toml

```diff
@@ -13,7 +13,7 @@ apollo-router-scaffold = { path ="{{integration_test}}apollo-router-scaffold" }
 {{#if branch}}
 apollo-router-scaffold = { git="https://github.com/apollographql/router.git", branch="{{branch}}" }
 {{else}}
-apollo-router-scaffold = { git="https://github.com/apollographql/router.git", tag="v1.1.0"}
+apollo-router-scaffold = { git="https://github.com/apollographql/router.git", tag="v1.2.0"}
```

Comment:
Not new in this PR, but should this be `apollo-router = "1.2.0"` from crates.io instead?

Reply:
Probably. Maybe raise an issue for this?